src/test/scala/mesosphere/marathon/api/akkahttp/v2/AppsControllerTest.scala

```diff
@@ -106,6 +106,18 @@ class AppsControllerTest extends UnitTest with GroupCreation with ScalatestRoute
       (body, plan)
     }
 
+    def addAppsToRootGroupRootGroup(apps: Seq[App]): (Seq[AppDefinition], DeploymentPlan, RootGroup) = {
```

Comment:
`addAppsToRootGroupRootGroup` is this like "FactoryManagerManager`? ^..^

Reply:
It seems that `preapreApp` and this one has a lot in common. Shouldn't one call the other (plus extras)? And shouldn't that be clear from their names?

Reply:
I don't know. Such helper methods in tests often point to issue in the actual code. Why is it so hard to add something to the root group?